docs/introduction/apphooks.rst

```diff
@@ -33,7 +33,9 @@ This is the most basic example of an apphook for a django CMS application:
 
     apphook_pool.register(PollsApphook)  # register the application
 
-Restart the runserver.
+**Restart the runserver**. This is necessary because we have created a new file containing Python
+code that won't be loaded until the server restarts. You only have to do this the first time the
+new file has been created.
```

Comment:
Here, we are not adding an apphook, but adding a new file containing an apphook. I don't believe that new files will ever get picked up until the runserver restarts.

So, an apphook re-configuration will restart the runserver, but only once the file containing the apphook has been picked up at least once by the system.